api/v1alpha1/release_types.go

```diff
@@ -173,7 +173,7 @@ func (r *Release) MarkInvalid(reason ReleaseReason, message string) {
 
 // MarkRunning registers the start time and changes the Succeeded condition to Unknown.
 func (r *Release) MarkRunning() {
-	if r.HasStarted() && r.Status.StartTime != nil {
+	if r.HasStarted() {
 		return
 	}
```

Comment:
FYI - This was removed because it appears `r.HasStarted()` already has the same conditional! So I thought it was redundant, but to be honest this could be out of scope for this patch? I'm writing unit tests, not trying to fix bugs or whatever. Was probably feeling frisky when I clobbered that conditional statement, and could be *way* out of line..

Regardless, here is the code for `HasStarted()`
```
// HasStarted checks whether the Release has a valid start time set in its status.
func (r *Release) HasStarted() bool {
	return r.Status.StartTime != nil && !r.Status.StartTime.IsZero()
}
```
You can see it has the conditional for not `nil` and not zero...

HOWEVER!
Upon further review, it appears `time.Time.IsZero()` also checks for nil.
```
// IsZero returns true if the value is nil or time is zero.
func (t *Time) IsZero() bool {
	if t == nil {
		return true
	}
	return t.Time.IsZero()
}
```
For reference: 
https://github.com/kubernetes/apimachinery/blob/v0.24.3/pkg/apis/meta/v1/time.go#L59

I haven't tested if `IsZero()` is sufficient on it's own, but it look like it might? 
We could potentially reduce code, further simplifying the testing. :crossed_fingers: